FearlessUtils/Classes/Crypto/BIP32ExtendedKeypair.swift

```diff
@@ -0,0 +1,24 @@
+import IrohaCrypto
+
+public class BIP32ExtendedKeypair {
```

Comment:
Consider to make `struct` because we don't use it as and object but as a value recreating each time to produce final keypair

Reply:
We cannot make it `struct` since it conforms to a class protocol `IRCryptoKeypairProtocol`